build.gradle.kts

```diff
@@ -1,38 +1,63 @@
 plugins {
-	id("maven-publish")
+	id("io.github.gradle-nexus.publish-plugin") version "1.0.0"
 	id("io.gitlab.arturbosch.detekt").version(Dependencies.detektVersion)
 }
 
 // Versioning
 allprojects {
 	group = "org.jellyfin.apiclient"
-	version = getProperty("jellyfin.version")?.removePrefix("v") ?: "SNAPSHOT"
+	version = getProperty("jellyfin.version")?.removePrefix("v") ?: "latest-SNAPSHOT"
 }
 
 buildscript {
-	repositories {
-		google()
-		jcenter()
-	}
+	repositories.defaultRepositories()
 
 	dependencies {
 		classpath(Dependencies.Android.buildTools)
 		classpath(Dependencies.Kotlin.gradlePlugin)
 	}
 }
 
-allprojects {
-	repositories {
-		google()
-		jcenter()
-	}
+// Add Sonatype publishing repository
+nexusPublishing.repositories.sonatype {
+	nexusUrl.set(uri("https://s01.oss.sonatype.org/service/local/"))
+	snapshotRepositoryUrl.set(uri("https://s01.oss.sonatype.org/content/repositories/snapshots/"))
+
+	username.set(project.properties["ossrh.username"].toString())
+	password.set(project.properties["ossrh.password"].toString())
+}
 
-	// Publishing
-	plugins.apply("maven-publish")
-	publishing.repositories.jellyfinBintray(this)
+subprojects {
+	// Enable required plugins
+	apply<MavenPublishPlugin>()
+	apply<SigningPlugin>()
+	apply<io.gitlab.arturbosch.detekt.DetektPlugin>()
```

Comment:
Could you do an import for detekt?

Reply:
I don't like imports in my build files 😅 (except for buildSrc)

Reply:
Ok then ^^